infra/docker/jobservice/Dockerfile

```diff
@@ -27,4 +27,10 @@ RUN wget -q https://github.com/grpc-ecosystem/grpc-health-probe/releases/downloa
          -O /usr/bin/grpc-health-probe && \
     chmod +x /usr/bin/grpc-health-probe
 
+ENV FEAST_SPARK_LAUNCHER standalone
+ENV FEAST_SPARK_STANDALONE_MASTER "local[*]"
+ENV FEAST_SPARK_HOME $SPARK_HOME
+ENV FEAST_SPARK_EXTRA_OPTIONS "--jars https://storage.googleapis.com/hadoop-lib/gcs/gcs-connector-hadoop2-latest.jar \
+--conf spark.hadoop.fs.gs.impl=com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem"
+
```

Comment:
Can we not hardcode those here and set them from docker-compose instead?

Reply:
I think that might break the GKE deployment too

Reply:
Is this still pending an update?

Reply:
I do think this shouldn't be here now that we have both gcp and aws, but I'm not comfortable making changes to this since I'm not able to test if things continue working on gcp. Also it's not just this local change, we need to look into how we configure helm charts as well, since there are bunch of gcp-related flags in helm charts where they shouldn't be. I think cleaning gcp infra setup to make it similar to current AWS setup (where everything aws-related is set from terraform) should be a separate piece of work.

Reply:
Are you saying that the Feast client doesn't work without these vars set? @pyalex can you confirm?

Reply:
@tsotnet What prevents us from
1. Adding this configuration to Docker Compose
2. Adding this configuration to a Helm values.yaml file (not the chart)

At least all tech debt is easy to find and clean up in the same place, right?

Reply:
Discussed this in the standup; resolving the conversation